### ISSUE TITLE
Recent Activity returning league does not exist for private football league

### ISSUE THREAD

**Issue by zanderman12** [cwendt94/espn-api#546]
### Sport

Football

### Summary

Trying to access the trade activity for my league. Have included the s2 and swid cookies when defining the league and am able to access league data (team records, player performance, etc.). However, when I try to request recent activity, I get: ESPNInvalidLeague: League ##### does not exist. I have tried this across multiple leagues with the same result. Using latest release version (0.36.0).  

ps. Thank you for all the hard work on this. It's incredibly useful!

**Comment by JaxonWright** [comment]
Can confirm that this is happening for me as well on 0.36.0

**Comment by danielobrien3** [comment]
I'm sadly not sure the historical data `recent_activities()` would pull from even exists anymore. If you go to your league homepage and select the drop down to look at another year's data, "recent activities" is no longer an option (or maybe never was, but I feel like this used to work). 

So it seems there has either been a breaking change with recent activities, or it will only return the current seasons data. Again, I don't really remember how this worked in the past.

I'd still love to be able to analyze transactions of past years, and the data exists in each player card (go back to whatever season, click a player, look at their transaction history), so maybe the data can still be scraped yet.

**Comment by cwendt94** [comment]
Yeah it does seem like they have removed the ability to see previous years recent activity.

I could definitely look at adding transaction history to the player data. I think it would have to be accessed through the `player_info` call but looks like they still include that data.

**Comment by samthom1** [comment]
For what it's worth, this is now broken for current year data, but still works for the 2024 year in my private hockey league.
